rosdep/python.yaml

```diff
@@ -5189,6 +5195,17 @@ python3-pytest:
   openembedded: [python3-pytest@meta-python]
   rhel: ['python%{python3_pkgversion}-pytest']
   ubuntu: [python3-pytest]
+python3-pytest-mock:
+  alpine: [py3-pytest-mock]
+  arch: [python-pytest-mock]
+  debian: [python3-pytest-mock]
+  fedora: [python3-pytest-mock]
+  gentoo: [dev-python/pytest-mock]
+  openembedded:
+    pip:
+      packages: [pytest-mock]
```

Comment:
I'm not sure that pip makes sense on openembedded it's not deployable as such there. Can you explain why you added it there with a pip rule?

Reply:
@tfoote I was trying to match pytest's footprint. pytest itself is available via rosdep on openembedded, but pytest-mock isn't in the layer index. Is this unnecessary?

Reply:
@tfoote I've removed it in f0b8bf2